packages/office-ui-fabric-react/src/components/Rating/Rating.tsx

```diff
@@ -7,28 +7,52 @@ import {
 import { Icon } from '../../Icon';
 import { IRatingProps, RatingSize } from './Rating.Props';
 import * as stylesImport from './Rating.scss';
+import { FocusZone, FocusZoneDirection } from '../../FocusZone';
 const styles: any = stylesImport;
 
+interface IRatingStarProps extends React.AllHTMLAttributes<HTMLElement> {
+  fillPercentage: number;
+  disabled: boolean;
+}
+
 export interface IRatingState {
   rating: number | null | undefined;
-  focusedRating: number | null;
 }
 
+const RatingStar = (props: any) => (
+  <div
+    className={ css('ms-RatingStar-container', styles.ratingStar) }
+    key={ props.id }
```

Comment:
why map id to key, can't we use `key`?

Reply:
props.key? doesn't pipe through. React strips it out.

Reply:
You're right... but good point, do we then need a key here at all?

Reply:
yes we need it as we are creating multiple stars . React throws error when no key is present

Reply:
yes, you are setting a key on `RatingStar` already.. there's where your array is.. Inside of `RatingStar` is only a single element, no need to a key. See line 93.